ml-agents-envs/mlagents/envs/base_unity_environment.py

```diff
@@ -9,9 +9,9 @@ class BaseUnityEnvironment(ABC):
     def step(
         self,
         vector_action: Optional[Dict] = None,
-        memory: Optional[Dict] = None,
         text_action: Optional[Dict] = None,
         value: Optional[Dict] = None,
+        custom_action: Dict[str, Any] = None,
```

Comment:
These are being passed (for now) by the implementations, but will likely go away soon.